src/components/Book.js

```diff
@@ -15,19 +17,43 @@ const Book = ({
   };
   return (
     <div className="book">
-      <h3>
-        Title:
-        {title}
-      </h3>
-      <p>
-        Author:
-        {author}
-      </p>
-      <p>
-        Category:
-        {category}
-      </p>
-      <RemoveBookButton type="button" onClick={handleRemoveClick}>Delete</RemoveBookButton>
+      <div className="book-cover">
+        <div className="book-info">
+          <h6 className="category">
+            {category}
+            Action
+          </h6>
+          <h3 className="title">
+            {title}
+          </h3>
+          <p className="author">
+            {author}
+          </p>
+        </div>
+        <div className="book-actions">
+          <p className="comments">
```

Comment:
- You have done great so far :+1: However there are a few changes you need to make before moving to the next milestone. :100: 

- [ ] Looking at the screenshot below, you can see that your book title and author tend to overlap each other. The author's name is on top of the title when the title is longer than 1 line. I'll suggest yo adjust the CSS to fix this. :smile: :wave: 

- - Screenshot reference:
 
![Selection_020](https://github.com/HossainAraf/Book-store/assets/107632780/a696a8a1-c1c2-4c85-8499-0cff582bef12)


- [ ] Also your design is missing the vertical line in-between the `completed` section and the `chapter details` section. I'll suggest you add that so that your finished design looks exactly like the provided design. :+1: :smile: